pkg/aggregator/mocksender/asserts.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package mocksender
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+)
+
+// AssertServiceCheck allows to assert a ServiceCheck was emitted with given parameters.
+// Additional tags over the ones specified don't make it fail
+func (m *MockSender) AssertServiceCheck(t *testing.T, checkName string, status metrics.ServiceCheckStatus, hostname string, tags []string, message string) bool {
```

Comment:
Why split `MockSender` methods in two file ? Can't those go with `pkg/aggregator/mocksender/mocksender.go` ?

Reply:
it's split in three files, with the dummy methods boilerplate in `mocked_methods.go`. I suspect I'll add lots more asserts as I go forward (events for starters), so I'm planning for future growth :)